### PR TITLE
feat - video listing page with category filter

### PR DESCRIPTION
**Pull request by prachi-sahani**
This PR includes video listing page along with a side nav
On click of particular category, users can view videos of that particular category

caketube - video listing

### CODE REVIEW

File: src/Components/sidenav/Sidenav.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+          <p className="txt txt-sm">Watch Later</p>
+        </li></Link>
+        <Link to="/likedVideos" className="link"><li className="list-item txt-center">
+          <i className="material-icons-outlined list-icon">thumb_up</i>
+          <p className="txt txt-sm">Liked Videos</p>
+        </li></Link>
+        <Link to="/history" className="link"><li className="list-item txt-center">
+          <i className="material-icons list-icon">history</i>
+          <p className="txt txt-sm">History</p>
+        </li></Link>
```
Comment (omjadhav85) [inline src/Components/sidenav/Sidenav.jsx]: Would suggest to create a sepetate component for SideBar items and use props for different titles and icons

File: src/Components/videoListing/VideoCard.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+          <button className="btn-icon btn-sm btn-outline-primary material-icons-outlined">
+            thumb_down
+          </button>
+          <button className="btn-icon btn-sm btn-outline-primary material-icons-outlined">
+            watch_later
+          </button>
+        </div>
+        {/* <button className="btn-basic btn-primary w-100 btn-watch-now">
+            Watch Now
+          </button> */}
```
Comment (omjadhav85) [inline src/Components/videoListing/VideoCard.jsx:30]: can remove comments if not required

**Review by omjadhav85 — APPROVED** [review]
Clean and easily understandable code. Some minor suggestion suggested. Great work!!

File: src/Components/sidenav/Sidenav.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { Link } from "react-router-dom";
+import "./sidenav.css"
+import "../../styles.css"
+export function Sidenav() {
+  return (
+    <div className="sidenav py-4 px-2">
+      <ul className="list-group-stacked">
+        <Link to="/" className="link"><li className="list-item txt-center">
```
Comment (junaidshaikh-js) [inline src/Components/sidenav/Sidenav.jsx]: `li` should be the direct child of `ul` nest `Link` inside `li`

File: src/Components/sidenav/sidenav.css [hunk 4]
```diff
@@ -0,0 +1,11 @@
+  
+  .sidenav {
+    /* width: 10rem; */
```
Comment (junaidshaikh-js) [inline src/Components/sidenav/sidenav.css]: code comments should be removed

File: src/Components/videoListing/VideoCard.jsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
+import "./videoListing.css";
+
+export function VideoCard({ video }) {
+  return (
+    <div className="video-card card card-w-badge">
+      <div className="card-image">
+        <img alt={video.title} className="card-img" src={video.thumbnail} />
+        <div className="product-rating">200 views</div>
+      </div>
```
Comment (junaidshaikh-js) [inline src/Components/videoListing/VideoCard.jsx:9]: You can use `figure` tag with `figcaption`  element

**Review by junaidshaikh-js — COMMENTED** [review]
Good work! 

Nice folder structure and clean code. Mentioned some points in the comments 

Everything else looks good.

File: src/Components/sidenav/Sidenav.jsx (hunk 3 above)
Comment (prachi-sahani) [reply src/Components/sidenav/Sidenav.jsx]: Done!

File: src/Components/sidenav/sidenav.css (hunk 4 above)
Comment (prachi-sahani) [reply src/Components/sidenav/sidenav.css]: Done!

File: src/Components/sidenav/Sidenav.jsx (hunk 1 above)
Comment (prachi-sahani) [reply src/Components/sidenav/Sidenav.jsx]: Done! Thank you :)

**Pull request merged**